galley/pkg/crd/validation/validation.go

```diff
@@ -43,8 +49,11 @@ func createMixerValidator() (store.BackendValidator, error) {
 }
 
 //RunValidation start running Galley validation mode
-func RunValidation(vc *WebhookParameters, printf, faltaf shared.FormatFn, kubeConfig string) {
-	//func RunValidation(vc *WebhookParameters,printf, faltaf shared.FormatFn,kubeConfig string){
+func RunValidation(vc *WebhookParameters, printf, faltaf shared.FormatFn, kubeConfig string,
+	livenessProbeController *probe.Controller, readinessProbeController *probe.Controller) {
+	defer func() {
+		validationProbe.SetAvailable(errors.New("stopped"))
+	}()
```

Comment:
This could be rewritten to be a bit shorter as readiness and liveness controllers share the same type:

```golang
func RunValidation(vc *WebhookParameters, printf, faltaf shared.FormatFn, kubeConfig string, livenessProbeController, readinessProbeController *probe.Controller) // nolint: lll
```

Also, I think you can defer the `SetAvailable()` function directly, e.g.

```golang
defer validationProbe.SetAvailable(errors.New("stopped"))
```

Reply:
done